packages/atomic/src/components/atomic-search-box/atomic-search-box.tsx

```diff
@@ -307,7 +318,7 @@ export class AtomicSearchBox {
         class="h-full outline-none bg-transparent flex-grow px-4 py-3.5 text-neutral-dark placeholder-neutral-dark text-lg"
         value={this.searchBoxState.value}
         onFocus={() => this.onFocus()}
-        onBlur={() => this.onBlur()}
+        onBlur={() => this.clearSuggestions()}
```

Comment:
On blur and refocus, would there now be a delay in seeing the suggestions while they are re-fetched?

Reply:
Yes, although it's not very noticeable even with a lot of throttling.